pkg/api/admin/operatorflags.go

```diff
@@ -0,0 +1,54 @@
+package admin
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+type operatorFlagsMergeStrategyStruct struct {
+	OperatorFlagsMergeStrategy string
+	Oc                         api.OpenShiftCluster
```

Comment:
The variable oc is a bit vague. It might be more clear to name it `cluster` or `OpenShiftCluster` ?

Reply:
Made the change.